LICENCE

```diff
@@ -0,0 +1,8 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
+ *
+ * This file is part of Agriculture-Core.
```

Comment:
Do you think we should have a more in depth description saying that anything in this project belongs to Boresha/JT?

Reply:
Hmm. That would probably be a good idea. I just used a template that I found online but I can add a few lines covering that.